<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>718</width>
    <height>471</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>File Encrypter/Decrypter</string>
  </property>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>721</width>
     <height>481</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QWidget#widget{
background-color:qlineargradient(spread:pad, x1:0.051, y1:0.0568182, x2:0.926136, y2:0.938, stop:0 rgba(100, 111, 162, 255), stop:1 rgba(218, 218, 218, 255))}</string>
   </property>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>350</y>
      <width>241</width>
      <height>81</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 22pt &quot;MS Shell Dlg 2&quot;;
color:rgb(0, 0, 0);</string>
    </property>
    <property name="text">
     <string>Select file</string>
    </property>
   </widget>
   <widget class="QPushButton" name="browse_button">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>430</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius:10px;
background-color: rgb(219, 212, 255);
font: 16pt &quot;MS Shell Dlg 2&quot;;</string>
    </property>
    <property name="text">
     <string>Browse</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="message_line">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>430</y>
      <width>331</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius:10px;
background-color: rgb(125, 124, 168);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="submit_button">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>430</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius:10px;
font: 16pt &quot;MS Shell Dlg 2&quot;;
background-color: rgb(219, 212, 255);</string>
    </property>
    <property name="text">
     <string>Send</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="ecb_radio_button">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>250</y>
      <width>95</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>ECB</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QRadioButton" name="cbc_radio_button">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>280</y>
      <width>95</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>CBC</string>
    </property>
   </widget>
   <widget class="QLabel" name="mode_text">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>180</y>
      <width>91</width>
      <height>81</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 22pt &quot;MS Shell Dlg 2&quot;;
color:rgb(0, 0, 0);</string>
    </property>
    <property name="text">
     <string>Mode</string>
    </property>
   </widget>
   <widget class="QLabel" name="ConnectionStatusLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>241</width>
      <height>81</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 22pt &quot;MS Shell Dlg 2&quot;;
color:rgb(0, 0, 0);</string>
    </property>
    <property name="text">
     <string>Disconnected</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="text_edit">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>40</y>
      <width>451</width>
      <height>311</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius:10px;
background-color: rgb(125, 124, 168);
font-size: 20px;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="connection_button">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>130</y>
      <width>241</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius:10px;
background-color: rgb(219, 212, 255);
font: 16pt &quot;MS Shell Dlg 2&quot;;</string>
    </property>
    <property name="text">
     <string>Connect/Disconnect</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="id_line">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>171</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius:10px;
background-color: rgb(125, 124, 168);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 22pt &quot;MS Shell Dlg 2&quot;;
color:rgb(0, 0, 0);</string>
    </property>
    <property name="text">
     <string>ID</string>
    </property>
   </widget>
   <widget class="QPushButton" name="id_button">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>40</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius:10px;
font: 16pt &quot;MS Shell Dlg 2&quot;;
background-color: rgb(219, 212, 255);</string>
    </property>
    <property name="text">
     <string>Set</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="receiver_id">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>360</y>
      <width>191</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius:10px;
background-color: rgb(125, 124, 168);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>320</y>
      <width>221</width>
      <height>111</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 15pt &quot;MS Shell Dlg 2&quot;;
color:rgb(0, 0, 0);</string>
    </property>
    <property name="text">
     <string>Receiver ID</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>350</y>
      <width>221</width>
      <height>121</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 15pt &quot;MS Shell Dlg 2&quot;;
color:rgb(0, 0, 0);</string>
    </property>
    <property name="text">
     <string>Message</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>-30</y>
      <width>221</width>
      <height>111</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 15pt &quot;MS Shell Dlg 2&quot;;
color:rgb(0, 0, 0);</string>
    </property>
    <property name="text">
     <string>Chat</string>
    </property>
   </widget>
   <widget class="QProgressBar" name="progress_bar">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>430</y>
      <width>118</width>
      <height>23</height>
     </rect>
    </property>
    <property name="value">
     <number>0</number>
    </property>
    <property name="textVisible">
     <bool>false</bool>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="keys_button">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>250</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius:10px;
background-color: rgb(219, 212, 255);
font: 16pt &quot;MS Shell Dlg 2&quot;;</string>
    </property>
    <property name="text">
     <string>Browse</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="password_line">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>340</y>
      <width>171</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius:10px;
background-color: rgb(125, 124, 168);</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::Password</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="password_button">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>340</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius:10px;
font: 16pt &quot;MS Shell Dlg 2&quot;;
background-color: rgb(219, 212, 255);</string>
    </property>
    <property name="text">
     <string>Set</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>300</y>
      <width>171</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 22pt &quot;MS Shell Dlg 2&quot;;
color:rgb(0, 0, 0);</string>
    </property>
    <property name="text">
     <string>Password</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
